pages/common/cppcheck.md

```diff
@@ -0,0 +1,32 @@
+# cppcheck
+
+> A static analysis tool for C/C++ code.
+> Instead of syntax errors, it focuses on the types of bugs that compilers normally do not detect.
+
+- Recursively check the current folder, showing progress on the screen and logging error messages to a file:
+
+`cppcheck . 2> cppcheck.log`
+
+- Recursively check a given folder, and don't print progress messages:
+
+`cppcheck --quiet {{path/to/folder}}`
+
+- Check a given file, with all tests enabled (by default only errors are shown) except two:
+
+`cppcheck --enable=all --suppress={{check_id1}} {{path/to/file.cpp}}`
```

Comment:
Maybe its better to say "test_id1" because you have "tests" in the description ? 

Also, you say "except two", but you suppress just one check. Is this okay ?